### PR TITLE
[plugin.video.dcmtv] 1.0.0

### PR DESCRIPTION
**Pull request by therezacuet**
### Checklist:

- [ ] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [ ] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [ ] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

File: plugin.video.dcmtv/addon.xml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.dcmtv"
+	name="DCM TV"
+	version="1.0.0"
+	provider-name="TonyTaliaferro">
+  <requires>
+    <import addon="xbmc.python" version="2.26.0"/>
+    <import addon="script.module.addon.common" version="2.0.0"/>
```
Comment (Rechi) [inline plugin.video.dcmtv/addon.xml]: This module is only used to assign one value to a variable which is never used.

File: plugin.video.dcmtv/addon.xml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  </requires>
+  <extension point="xbmc.python.pluginsource" library="default.py">
+    <provides>video</provides>
+  </extension>
+  <extension point="xbmc.addon.metadata">
+    <summary lang="en">DCM TV Add-on</summary>
+    <description lang="en">DCM TV features Originals, Clasic tv, Urban films and documentaries from around the world enjoy Independent films from many talented filmmakers.</description>
+    <platform>all</platform>
+    <source>https://github.com/therezacuet/kodi-tv-youtube-channel-ad-on</source>
+    <website>http://thereza.herokuapp.com</website>
```
Comment (Rechi) [inline plugin.video.dcmtv/addon.xml]: Website should point to the content website, not to your personal one.

File: plugin.video.dcmtv/default.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import os
+import sys
+import plugintools
+import xbmc,xbmcaddon
+from addon.common.addon import Addon
```
Comment (Rechi) [inline plugin.video.dcmtv/default.py]: This import is only used to assign one value to a variable which is never used.

File: plugin.video.dcmtv/default.py [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+addonID = 'plugin.video.dcmtv'
+addon = Addon(addonID, sys.argv)
```
Comment (Rechi) [inline plugin.video.dcmtv/default.py]: `addon` variable is unused.

File: plugin.video.dcmtv/default.py [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+local = xbmcaddon.Addon(id=addonID)
+icon = local.getAddonInfo('icon')
+YOUTUBE_CHANNEL_ID = "MrMrtonyangelo"
+def run():
+    plugintools.log("dcmtv.run")
```
Comment (Rechi) [inline plugin.video.dcmtv/default.py]: Kodi logs that already.

File: plugin.video.dcmtv/default.py [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+    params = plugintools.get_params()
+    if params.get("action") is None:
+        main_list(params)
+    else:
+        pass
+    plugintools.close_item_list()
+def main_list(params):
+    plugintools.log("dcmtv.main_list "+repr(params))
```
Comment (Rechi) [inline plugin.video.dcmtv/default.py]: Kodi logs that already.

File: plugin.video.dcmtv/plugintools.py [hunk 7]
```diff
@@ -0,0 +1,582 @@
...
+    skin_name = xbmc.getSkinDir()
+    _log("set_view skin_name='"+skin_name+"'")
+
+    try:
+        if view_code==0:
+            _log("set_view view mode is "+view_mode)
+            view_codes = ALL_VIEW_CODES.get(view_mode)
+            view_code = view_codes.get(skin_name)
+            _log("set_view view code for "+view_mode+" in "+skin_name+" is "+str(view_code))
+            xbmc.executebuiltin("Container.SetViewMode("+str(view_code)+")")
```
Comment (Rechi) [inline plugin.video.dcmtv/plugintools.py]: This isn't allowed.

File: plugin.video.dcmtv/plugintools.py [hunk 8]
```diff
@@ -0,0 +1,582 @@
...
+        else:
+            _log("set_view view code forced to "+str(view_code))
+            xbmc.executebuiltin("Container.SetViewMode("+str(view_code)+")")
```
Comment (Rechi) [inline plugin.video.dcmtv/plugintools.py]: This isn't allowed.

File: plugin.video.dcmtv/plugintools.py [hunk 9]
```diff
@@ -0,0 +1,582 @@
...
+        xlistitem = xbmcgui.ListItem( title, iconImage="DefaultVideo.png", )
+    xlistitem.setInfo( "video", { "Title": title } )
+
+    playlist = xbmc.PlayList( xbmc.PLAYLIST_VIDEO )
+    playlist.clear()
+    playlist.add( url, xlistitem )
+
+    player_type = xbmc.PLAYER_CORE_AUTO
+    xbmcPlayer = xbmc.Player( player_type )
+    xbmcPlayer.play(playlist)
```
Comment (Rechi) [inline plugin.video.dcmtv/plugintools.py]: This isn't allowed for plugins.

**Comment by enen92** [comment]
ping @therezacuet

**Comment by enen92** [comment]
Closed as pr is stalled. Resubmit if the interest persists

**Pull request closed without merging**